tensorboard/plugins/mesh/mesh_plugin_test.py

```diff
@@ -61,10 +64,11 @@ def setUp(self):
                                                     point_cloud.vertices.shape)
 
     mesh_no_color = test_utils.get_random_mesh(2000, add_faces=True)
+    mesh_no_color_upd = test_utils.get_random_mesh(2500, add_faces=True)
```

Comment:
`mesh_no_color_upd`: what does `upd` mean? Shortening variable for non-obvious case impairs readability.